compiler/src/main/resources/play_scala_security_extractors.mustache

```diff
@@ -8,6 +8,11 @@ import de.zalando.play.controllers.SwaggerSecurityExtractors._
 import {{import.name}}
 {{/for}}
 
+object SecurityExtractorsExecutionContext {
+    // TODO override with proper ExecutionContext instance
+    implicit val ec = scala.concurrent.ExecutionContext.global
```

Comment:
Shouldn't the default be the default Play execution context instead of the Scala global execution context?
http://stackoverflow.com/a/30806548